packages/boards/sender/platformio_upload.py

```
@@ -0,0 +1,143 @@
+#
+# An example of an upload URL:
+#                custom_upload_url = http://192.168.1.123/update
+# also possible: custom_upload_url = http://domainname/update
+
+import sys
+import requests
+import hashlib
+from urllib.parse import urlparse
+import time
+from requests.auth import HTTPDigestAuth
+Import("env")
```

Ruff (0.12.2)

Undefined name `Import`

(F821)